test/browser/package.json

```diff
@@ -9,6 +9,7 @@
     "test:playwright": "PROVIDER=playwright pnpm run test:unit",
     "test:safaridriver": "PROVIDER=webdriverio BROWSER=safari pnpm run test:unit",
     "test-fixtures": "vitest",
+    "test-expect-dom": "vitest --root ./fixtures/expect-dom",
```

Comment:
Is this suite run on CI?

Reply:
Oh! I forgot to add the CI one, was only running this locally 😄